google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineService.java

```diff
@@ -125,37 +124,30 @@
    protected synchronized void cleanUpIncidentalResourcesOfDeadNodes(Set<? extends NodeMetadata> deadNodes) {
       Set<String> orphanedGroups = findOrphanedGroups.apply(deadNodes);
       for (String orphanedGroup : orphanedGroups) {
-         cleanUpNetworksAndFirewallsForGroup(orphanedGroup);
+         cleanUpFirewallsForGroup(orphanedGroup);
```

Comment:
this code never gets called, I suspect there is a bug in BaseComputeService, I'll let you know soon